rules/TypeDeclaration/TypeAnalyzer/AlwaysStrictScalarExprAnalyzer.php

```diff
@@ -119,32 +112,4 @@ private function resolveTypeFromScalar(Scalar $scalar): Type|null
         return null;
     }
 
-    private function resolveNativeFuncCallType(FuncCall $funcCall, Scope $scope): Type
```

Comment:
this stuff is already done when we call into `nodeTypeResolver->getNativeType()`